### ISSUE TITLE
ARIA should be a last resort

### ISSUE THREAD

**Issue by oliverjam** [fac24/week2-Bereket-Hallie-Joe-Petra#37]
ARIA should generally be used as a last resort if the page cannot be made accessible without it. In this case all the form elements have visible labels, so adding extra `aria-label`s is at best redundant and at worst confusing for screen reader users.

https://github.com/fac24/week2-Bereket-Hallie-Joe-Petra/blob/2404dd6fac72e98f1756473191fa85bf13a99bc1/routes/addReview.js#L10-L11

would be a bit weird for a user who could see _and_ uses a screen reader (most legally blind people still have some vision). They would _see_ "Username" but hear something different and more verbose.

Inputs implicitly already tell a user to "insert something". If you wouldn't include instructions for a visual user you probably don't need to add it for a non-visual one. It's similar to not putting "image of an x" in alt text—they know it's an image from the `<img>` bit 😅 

Assuming your visible labels are correctly connected to the input (with `for="id"`) then assistive tech should figure out the label fine without any ARIA at all.
